8.cytopick_analysis/notebooks/find_prediction_images.ipynb

```diff
@@ -1,1914 +1,1911 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        # select the top n cells
</pre><p>Does the function assume that the dataframe is sorted, if so, consider addeding that to the documentation or add a line that conducts the sorting within the function. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>addressed</p>